packages/block-editor/src/components/rich-text/get-rich-text-values.js

```diff
@@ -40,7 +40,9 @@ function addValuesForElement( element, ...args ) {
 		case Fragment:
 			return addValuesForElements( props.children, ...args );
 		case RawHTML:
-			return;
+			// `useInnerBlocksProps.save()` will return a `RawHTML` element,
+			// so use this as an indicator to recurse into the children.
+			return addValuesForBlocks( ...args );
```

Comment:
How would you know that this is the correct RawHTML instance for which to process inner blocks?